core/templates/dev/head/domain/topic/SubtopicPageObjectFactorySpec.ts

```diff
@@ -16,13 +16,28 @@
  * @fileoverview Tests for SubtopicPageObjectFactory.
  */
 
+// TODO(YashJipkate) Remove the following block of unnnecessary imports once
```

Comment:
Follow the `TODO(username): xxx` pattern here any everywhere else.

Reply:
I think this would be best suited in a different PR since the previously merged PR had this convention too and changing them would bringing unnecessary diffs. Actually, I saw [this](https://github.com/oppia/oppia/blob/develop/core/templates/dev/head/domain/objects/UnitsObjectFactory.ts#L31) and applied it.

Reply:
Never mind, done!